lang/cs/Org.Apache.REEF.Network.Tests/GroupCommunication/GroupCommunicationTests.cs

```diff
@@ -160,6 +160,63 @@ public void TestBroadcastReduceOperators()
         }
 
         /// <summary>
+        /// Test create a new group then remove it from the GroupDriver
+        /// </summary>
+        [Fact]
+        public void TestRemoveCommunicationGroup()
+        {
+            string groupName = "group1";
+            string groupName2 = "group2";
+            string masterTaskId = "task0";
+            string driverId = "Driver Id";
+            int numTasks = 3;
+            int fanOut = 2;
+
+            var groupCommunicationDriver = GetInstanceOfGroupCommDriver(driverId, masterTaskId, groupName, fanOut, numTasks);
+            groupCommunicationDriver.NewCommunicationGroup(groupName2, 5);
+            groupCommunicationDriver.RemoveCommunicationGroup(groupName2);
```

Comment:
How do you know that group was actually removed. I would try to remove it again, catch the ArgumentNull exception and test if its message is the expected one